python/src/worker/tasks/differential_expression.py

```diff
@@ -112,12 +113,13 @@ def compute(self):
         request = {
             "baseCells": [int(x) for x in first_cell_set],
             "backgroundCells": [int(x) for x in second_cell_set],
-            "pagination": self.pagination,
         }
 
-        if "filters" in self.pagination:
-            gene_filter = self.pagination["filters"][0]["expression"]
-            request["geneNamesFilter"] = remove_regex(gene_filter)
+        if self.pagination:
```

Comment:
{} returns False in python

Reply:
you don't need this if statement, because in line 25 you are already setting `pagination` to `{}`, so it can never be undefined. Now, if `pagination` is an empty object, then `if "filters" in self.pagination` will just return false.

Reply:
however, if pagination is empty it will not be added to the request. not sure if that's intended or not but it wont' be equivalent

Reply:
hmm, that's a good point. No idea if intended or not